corehq/apps/analytics/tasks.py

```diff
@@ -340,7 +340,7 @@ def submit_data_to_hub_and_kiss(submit_json):
         try:
             dispatcher(submit_json)
         except Exception, e:
-            notify_exception(None, "{msg}: {exc}".format(msg=error_message, exc=str(e)))
+            notify_exception(None, unicode("{msg}: {exc}".format(msg=error_message, exc=e)))
```

Comment:
I don't think this will work as the `.format()` call will still fail. what you want is to just prefix the string itself with a u.
```python
>>> delhi = u'दिल्ली'
>>> delhi
u'\u0926\u093f\u0932\u094d\u0932\u0940'
>>> 'I am in {}'.format(delhi)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)
>>> u'I am in {}'.format(delhi)
u'I am in \u0926\u093f\u0932\u094d\u0932\u0940'
```